docs/qcreset.rst

```diff
@@ -0,0 +1,218 @@
+The qcreset module
+=====================
```

Comment:
`pip install rstcheck` and then run `rstcheck` on this file will give you some details to fix :)

(I will add `rstcheck`to the CI pipeline eventually)